### PR TITLE
Fix CICS Enter Tracenum Command

### PR DESCRIPTION
**Pull request by KutluOzel-b**
## Checklist:
- [x] Each of my commits contains one meaningful change
- [x] I have performed rebase of my branch on top of the development
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by KutluOzel-b** [comment]
@chacebot @mm-broadcom @ilidio-lopes Can you please review ?

File: server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSEnterTracenumOptionsCheckUtility.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    private static final Map<Integer, ErrorSeverity> DUPLICATE_CHECK_OPTIONS =
+            new HashMap<Integer, ErrorSeverity>() {
+                {
+                    put(CICSLexer.ENTER, ErrorSeverity.ERROR);
+                    put(CICSLexer.TRACENUM, ErrorSeverity.ERROR);
+                    put(CICSLexer.FROM, ErrorSeverity.ERROR);
+                    put(CICSLexer.FROMLENGTH, ErrorSeverity.ERROR);
+                    put(CICSLexer.RESOURCE, ErrorSeverity.ERROR);
+                    put(CICSLexer.EXCEPTION, ErrorSeverity.ERROR);
```
Comment (ilidio-lopes) [inline server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSEnterTracenumOptionsCheckUtility.java]: Warning for EXCEPTION

Comment (ilidio-lopes) [reply server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSEnterTracenumOptionsCheckUtility.java]: has conflicts

**Comment by ilidio-lopes** [comment]
also failing tests

**Review by ilidio-lopes — APPROVED** [review]

File: server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCicsEnterTracenumStatement.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ * <p>This class tests all variations of the Enter Tracenum command found in the link above.
+ */
+public class TestCicsEnterTracenumStatement {
+    private static final String ENTER_TRACENUM_VALID =
+            "ENTER TRACENUM({$varFour})";
+    private static final String ENTER_TRACENUM_ALL_VALID =
+            "ENTER TRACENUM({$varFour}) FROM({$varFour}) \n"
+                   + "FROMLENGTH({$varFour}) RESOURCE({$varFour}) EXCEPTION";
+    private static final String ENTER_TRACENUM_FROMLENGTH_INVALID =
+            "ENTER  {_TRACENUM(100) FROMLENGTH(100)|errorMissingFrom_}";
```
Comment (slavek-kucera) [inline server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCicsEnterTracenumStatement.java]:
```
DFH7226I           E        00001         SUB-OPERAND '100' OF 'FROMLENGTH' KEYWORD SHOULD BE A DATA REFERENCE NOT AN
                                          EXPRESSION OR A CONSTANT. COMMAND NOT TRANSLATED.                          
```

Comment (KutluOzel-b) [reply server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCicsEnterTracenumStatement.java]: We didn't validate sub-operand data type in any test of the CICS implementations. Is the decision has changed ?

Comment (slavek-kucera) [reply server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCicsEnterTracenumStatement.java]: This is not about adding new validation in the PR - this is about modifying the test case so there is no extra discrepancy from compilation on MF.
The test case is supposed to be invalid due to a missing operand only.
If/when operand value validation is added in the future, this test will start failing.

Comment (KutluOzel-b) [reply server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCicsEnterTracenumStatement.java]: I'm pointing out that for all CICS related fixes we need to cover this if we do this on this PR.

**Review by slavek-kucera — APPROVED** [review]

**Pull request merged**